src/hotspot/cpu/aarch64/stubDeclarations_aarch64.hpp

```diff
@@ -44,7 +44,7 @@
                                        do_arch_blob,                    \
                                        do_arch_entry,                   \
                                        do_arch_entry_init)              \
-  do_arch_blob(compiler, 30000 ZGC_ONLY(+10000))                        \
+  do_arch_blob(compiler, 35000 ZGC_ONLY(+5000))                        \
```

Comment:
Alignment of `\`

Reply:
Fixed here and in final blob declaration